modules/virt-about-vtpm-devices.adoc

```diff
@@ -18,6 +18,16 @@ the node has one.
 
 A vTPM device also protects virtual machines by storing secrets without physical hardware. {VirtProductName} supports persisting vTPM device state by using Persistent Volume Claims (PVCs) for VMs. You must specify the storage class to be used by the PVC by setting the `vmStateStorageClass` attribute in the `HyperConverged` custom resource (CR):
 
+
+[IMPORTANT]
+====
+**vTPM Storage class and live migration considerations**
+
+In {VirtProductName}, if the `storageClass` for `vmStateStorageClass` is not explicitly defined, the system uses the cluster’s default storage class to create the vTPM persistent volume claim (PVC). The access mode for the PVC is determined by the storage profile of the storage class.
```

Comment:
PVC was already defined earlier in the module:

```suggestion
In {VirtProductName}, if the `storageClass` for `vmStateStorageClass` is not explicitly defined, the system uses the cluster’s default storage class to create the vTPM PVC. The access mode for the PVC is determined by the storage profile of the storage class.
```

Reply:
It occurs to me that the second paragraph of this admonition is the really important part and we may be distracting from it by including the other information upfront. I would suggest taking the first paragraph out and putting it in the normal body text.